[Huh?](https://travis-ci.org/mathiasbynens/regenerate-unicode-properties/jobs/208726335#L899)

```
Creating Binary_Property/Emoji_Component.js…
module.js:472
    throw err;
    ^
Error: Cannot find module 'unicode-tr51/Emoji_Component.js'
```

--- a/package.json
+++ b/package.json
@@ -38,7 +38,7 @@
     "fs-extra": "^1.0.0",
     "jsesc": "^2.2.0",
     "unicode-9.0.0": "^0.7.0",
-    "unicode-tr51": "^7.0.2"
+    "unicode-tr51": "^8.0.1"
   },
   "scripts": {
     "build": "node build.js",

--- a/tests/tests.js
+++ b/tests/tests.js
@@ -9,6 +9,9 @@ test(t => {
 	t.true(
 		require('../Binary_Property/Emoji.js') instanceof regenerate
 	);
+	t.true(
+		require('../Binary_Property/Emoji.js').toRegExp().test('\u{1F921}') // U+1F921 CLOWN FACE
+	);
 	t.throws(
 		() => require('../Invalid_Property/X.js'),
 		Error

